tests/common.py

```diff
@@ -400,7 +400,7 @@ def setUp(self):
     super().setUp()
     self.settings_mods = {}
     self.emcc_args = ['-Werror']
-    self.node_args = ['--stack-trace-limit=50', '--unhandled-rejections=throw']
+    self.node_args = ['--stack-trace-limit=50', '--trace-uncaught', '--unhandled-rejections=throw']
```

Comment:
unintended?

Reply:
Its somewhat unrelated but also makes node error message more useful in general.  See https://nodejs.org/api/cli.html#cli_trace_uncaught.  Basically without this node prints stack traces for uncaught Error object but not the full stack trace for other kinds of objects.   I could put this in a separate PR if you prefer but I think its OK here.

Reply:
I guess it's ok here. It does say it may make things slower though, in that link? Is it worth running all the time then?

Reply:
I'll revert this here and propose it as a separate PR

Reply:
https://github.com/emscripten-core/emscripten/pull/14890